core/src/plan/expr/mod.rs

```diff
@@ -69,6 +69,11 @@ impl<'a> From<&'a Expr> for PlanExpr<'a> {
                 let exprs = indexes.iter().chain(once(obj.as_ref())).collect();
                 PlanExpr::MultiExprs(exprs)
             }
+            Expr::Array { elem } => {
+                let exprs = elem.iter().collect();
+                PlanExpr::MultiExprs(exprs)
+                //fix
```

Comment:
we can remove this unused comment

Reply:
Oh, i forgot to remove following, thx for pointing this out